src/img/menvcfgmodereg.edn

```diff
@@ -4,13 +4,12 @@
 (def row-height 45)
 (def row-header-fn nil)
 (def boxes-per-row 32)
-(draw-column-headers {:height 20 :font-size 18 :labels (reverse ["0" "" "1" "3" "4" "5" "6" "" "7" "" "8" "" "" "" "" "" "27" "28" "" "29" "" "30" "" "" "" "" "" "61" "" "62" "" "63"])})
+(draw-column-headers {:height 20 :font-size 18 :labels (reverse ["0" "" "1" "3" "4" "5" "6" "" "7" "" "8" "" "" "" "" "" "" "27" "28" "" "" "" "" "" "" "" "" "61" "" "62" "" "63"])})
 
 (draw-box "STCE" {:span 2})
 (draw-box "PBMTE" {:span 2})
-(draw-box (text "WPRI" {:font-weight "bold"}) {:span 7})
+(draw-box (text "WPRI" {:font-weight "bold"}) {:span 9})
```

Comment:
This isn't rendering correctly

<img width="569" alt="image" src="https://github.com/riscv/riscv-cheri/assets/59061376/eeee8cab-18a4-4f14-829e-af32d8aea6d4">

and it's only 1 bit in the text, not 2